packages/driver/src/cy/focused.js

```diff
@@ -91,7 +91,7 @@ const create = (state) => {
 
     // store the current focused element
     // since when we call .focus() it will change
-    const $focused = getFocused()
+    const $focused = getFocused(el.ownerDocument)
```

Comment:
Add link to issue this is fixing. Would be better to have a comment explaining why we have to send ownerDocument here.

I think you'll want to call `$document.getDocumentFromElement(el)` imported from `./document` since this has some extra checks around if the el is the actual document. https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/dom/document.ts#L30:L30

```suggestion
    // https://github.com/cypress-io/cypress/issues/8111
    const $focused = getFocused($document.getDocumentFromElement(el))
```